service/src/main/java/bio/terra/pipelines/iam/AuthenticatedUserRequest.java

```diff
@@ -0,0 +1,94 @@
+package bio.terra.pipelines.iam;
+
+import bio.terra.common.exception.ApiException;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AuthenticatedUserRequest {
```

Comment:
this isn't being used now - can we delete these two classes for now? (this and AuthenticatedUserRequestFactory.java)